algorithms/unit_tests/CMakeLists.txt

```diff
@@ -23,21 +23,15 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;HIP;SYCL;OpenMPTarget)
     # Generate a .cpp file for each one that runs it on the current backend (Tag),
     # and add this .cpp file to the sources for UnitTest_RandomAndSort.
     set(ALGO_SORT_SOURCES)
-    foreach(SOURCE_Input
-	TestSort
-	TestSortByKey
-	TestSortCustomComp
-	TestBinSortA
-	TestBinSortB
-	TestNestedSort
-      )
+    foreach(SOURCE_Input TestSort TestSortByKey TestSortCustomComp TestBinSortA
+                         TestBinSortB TestNestedSort
+    )
```

Comment:
I guess it's really a question of taste, but I don't find this one-line list of arguments easy to read.